.eslintrc.js

```diff
@@ -21,8 +21,12 @@ module.exports = {
     'mocha/no-skipped-tests': 'error',
     'mocha/no-synchronous-tests': 'error',
     'mocha/no-top-level-hooks': 'error',
-    'mocha/valid-suite-description': ['warn', '^[A-Z#]'],
-    'mocha/valid-test-description': 'warn',
+    'mocha/valid-suite-description': ['warn', /^[A-Z#]/],
```

Comment:
What are these changes for?

Reply:
for suit cases message:
```js
it('[getId] should return string'
```

Reply:
for consistency - `/ /` - is regexp expression